variables.tf

```diff
@@ -79,3 +85,44 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "storageClass" {
+  type        = string
+  description = "The name of the server"
+}
+
+variable "database_name" {
+  type        = string
+  description = "The name of the database to be created"
+  default     = "OMS_DB"
+}
+
+variable "db2instancetype" {
+  type        = string
+  description = "type of db2 intance to be created - db2oltp of db2wh"
+  default     = "db2oltp"
+}
+
+variable "db2instanceversion" {
+  type        = string
+  description = "version of the DB2 instance"
+  default     = "11.5.5.0-x86_64"
+}
+
+variable "defaultuserpwd" {
+  type        = string
+  description = "Password of the default user"
+  default     = "db2password"
+}
+
+variable "db2instanceid" {
```

Comment:
Where does this value come from? Is it looked up from somewhere? Is it made up?

Reply:
Planning to create it randomly using random_id(https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/id)